gulp_tasks/test.js

```diff
@@ -0,0 +1,7 @@
+var gulp = require('gulp');
+var mocha = require('gulp-mocha');
+
+gulp.task('test', function() {
+    gulp.src(['test/setup/unit.js', 'src/**/_unit_/*_test.js'])
```

Comment:
you got rid of test/**/* but you left this line in.  are we going to keep chai / sinon chai / mockery / nock?